cypress/e2e/swap.test.ts

```diff
@@ -431,16 +312,12 @@ describe('Swap', () => {
           // Open the currency select modal.
           cy.get('#swap-currency-output .open-currency-select-button').click()
 
-          // Wait for the currency list to load
-          cy.contains('1inch').should('exist')
-
           // Select UNI as output token
           cy.get(getTestSelector('token-search-input')).clear().type('Uniswap')
           cy.get(getTestSelector('currency-list-wrapper'))
             .contains(/^Uniswap$/)
             .first()
-            .should('exist')
-            .click()
+            .click({ force: true })
```

Comment:
This was flaky because our scrolling library was mid-scroll when cypress kicked in, so there was a `pointer-events: none` on the element. I got around it with `force: true`.